docs/src/main/sphinx/admin/properties-client-protocol.md

```diff
@@ -0,0 +1,183 @@
+# Client protocol properties
+
+The following sections provide a reference for all properties related to the
+[client protocol](/client/client-protocol). 
+
+(prop-protocol-spooling)=
+## Spooling protocol
+
+The following properties are related to the [](protocol-spooling).
+
+### `protocol.spooling.enabled`
+
+- **Type:** [](prop-type-boolean)
+- **Default value:** `true` (is that automatically configured if shares secret or so is set up)
+
+Enable spooled client protocol support. 
+
+### `protocol.spooling.shared-secret-key`
+
+- **Type:** [](prop-type-string)
+
+256 bit, base64-encoded secret key used to secure segment identifiers.
+
+### `protocol.spooling.retrieval-mode`
+    
+- **Type:** [](prop-type-string)
+- **Default value:** `STORAGE`
+
+Determines how the client retrieves the segment. Following are possible values:
+
+* `STORAGE` - client accesses the storage directly with the pre-signed URI, 1
+  round trip
+* `COORDINATOR_STORAGE_REDIRECT` - client accesses the coordinator and is
+  redirected to the storage with the pre-signed URI, 2 round trips
+* `COORDINATOR_PROXY` - client accesses the coordinator and gets segment data
```

Comment:
client accesses the coordinator which returns segment data fetched from the storage (1 http request per segment)